docs/spec/nft/02_state.md

```diff
@@ -0,0 +1,20 @@
+# State
+
+## Collections
+
+As all NFTs belong to a specific `Collection`, they are kept on store in an array
+within each `Collection`. Every time an NFT that belongs to a collection is updated,
+it needs to be updated on the corresponding NFT array on the corresponding `Collection`.
```

Comment:
whoa - are you saying NFTs are stored in two places? that sounds like asking for trouble. We should most certainly only store pointer objects in the collection to avoid having to update both. (See my suggested code change to the spec).